execution/executor/src/components/chunk_commit_queue.rs

```diff
@@ -16,10 +16,11 @@ pub struct ChunkCommitQueue {
 
 impl ChunkCommitQueue {
     pub fn new_from_db(db: &Arc<dyn DbReader>) -> Result<Self> {
-        let persisted_view = db
-            .get_startup_info()?
-            .ok_or_else(|| anyhow!("DB not bootstrapped."))?
-            .into_latest_executed_trees();
+        let persisted_view = match db.get_startup_info()? {
+            Some(info) => info.into_latest_executed_trees(),
+            // This is a hack for backup-types tests.
+            None => db.get_latest_executed_trees()?,
+        };
```

Comment:
what's the difference between this and just calling `db.get_latest_executed_trees()`?

Reply:
no diff!